### PR TITLE
add 2015 and 2016

### PR DESCRIPTION
**Pull request by butlerx**
2015 is good but 2016 is the most up to date json file i could find and its not right

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -65,10 +65,10 @@ setInterval(function(){
   var mins = Math.floor( (dist % _hour) / _minute );
   var secs = Math.floor( (dist % _minute) / _second );
 
-  var str  = 	(days > 0) 		? "<span>" + days + " Days</span>" 		: "";
-  str +=		(hours > 0) 	? "<span>" + hours + " Hours</span>"	: "";
-  str +=		(mins > 0) 		? "<span>" + mins + " Minutes</span>"	: "";
-  str +=		"<span>" + secs + " Seconds</span>";
+  var str  = 	(days > 0) 		? '<span>' + days + ' Days</span>' 		: '';
```
Comment (SeanHealy33) [inline js/main.js]: this might be the most unreadable delta I've seen in a while 😉

File: 2015/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<!--
+#######################################
+# 			Techweek 2015             #
```
Comment (SeanHealy33) [inline 2015/index.html]: pedantic but can you make these inline my ocd is off the richter scale.

File: 2016/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<!--
+#######################################
+# 			Techweek 2015             #
```
Comment (SeanHealy33) [inline 2016/index.html]: ⚡

File: 2015/index.html (hunk 2 above)
Comment (butlerx) [inline 2015/index.html]: i dont even care abut this thing

File: js/main.js (hunk 1 above)
Comment (butlerx) [inline js/main.js]: linting :P

**Pull request merged**